analysers/analyser_merge_bicycle_parking_FR_paris.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2014-2016                                 ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+STANDS_TYPES = {
+    "Arceau autre": "stands",
+    "Arceau St-Germain": "wide_stands",
+    "Epingle vélo": "stands",
+    "Sans": "floor",
+}
+tag_mapping = {
+    "capacity": (
+        lambda res: None if res["PLACAL"] in (None, "0") else res["PLACAL"]
+    ),
+    "bicycle_parking": (
+        lambda res: STANDS_TYPES.get(res['TYPEMOB'], None)
+    )
+}
+
+
+class Analyser_Merge_Bicycle_Parking_FR_Paris(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        self.missing_official = {"item":"8150", "class": 1, "level": 3, "tag":
```

Comment:
Since we already have a parking item we can reuse it:
http://osmose.openstreetmap.fr/fr/errors/?item=8150
Bu we need to change class, replace 1 by 21.